### DISCUSSION TITLE
[BUG] switching discussion category deletes the text

### DISCUSSION THREAD

**Discussion by d-schmidt** [community/community#7964] [Discussions] [answered]
I lost half a page of text because I didn't select a category before I started typing. Why is this text deleted when I switch category?

![grafik](https://user-images.githubusercontent.com/3226483/142763574-2b6b55c4-e5e6-4bc2-bcea-a08d8d719676.png)

**Comment by ls-andrew-borstein** [comment]
I just had this happen to me and it is insanely frustrating. I spent a half hour writing content in the main textarea only to have it vanish.

**Comment by ls-andrew-borstein** [comment]
**Update**: I discovered the contents of the textarea may be preserved in the DOM if you inspect the page. The blurred section below is all the of the text I had entered — you can see the textarea is empty given the placeholder prompt.

![Screen Shot 2021-11-22 at 12 54 00 PM](https://user-images.githubusercontent.com/82463438/142911940-85a167c8-3a53-493f-a97f-b54691dba5ae.png)

I don't think it will persist on page refresh, though.

**Reply by jaredkrinke** [reply]
FYI: It did persist for me on page refresh (in Edge), but yes, F12 dev tools is the way to get your text back.

**Comment by Andre601** [comment]
Can reporoduce this issue.
It's quite annoying when you write a rather large text, either to announce something or for simple asking, and then you loose that text because you didn't select the category first...

I hope GitHub fixes this **fast**.

**Comment by 0Grit** [comment]
I've been bit by this twice. Really frustrating.

**Comment by jaredkrinke** [comment, marked as answer]
Switching categories *did not* delete my title and text today.

**Did this get fixed?**

**Reply by d-schmidt** [reply]
seem like it
